classes/scripts/runner.php

```diff
@@ -247,7 +247,7 @@ public function useConfigurationCallable(\closure $callback)
         $runner = $this->runner;
         $errors = [];
 
-        $this->adapter->set_error_handler(function ($error, $message, $file, $line, $context) use (&$errors) {
+        $this->adapter->set_error_handler(function ($error, $message, $file, $line, $context = []) use (&$errors) {
```

Comment:
why the $context argument have a default value, now?

Reply:
It has been removed and an error is thrown in case of error on PHP8 if it remains a mandatory parameter.

Source: https://github.com/php/php-src/blob/master/UPGRADING
`  . Removed the $errcontext argument for custom error handlers.`

Reply:
thanks,